src/Accordion/index.jsx

```diff
@@ -60,25 +56,16 @@ export default class Accordion extends Component {
       return null;
     }
 
-    if (!Array.isArray(this.props.children)) {
-      const expanded = !this.props.disabled && this.state.activeItems.indexOf(0) !== -1;
-
-      return React.cloneElement(this.props.children, {
-        expanded: expanded,
-        key: 0,
-        onClick: this.handleClick.bind(this, 0, this.props.children.props.onClick),
-        ref: `item-${ 0 }`
-      });
-    }
-
-    return this.props.children.map((item, index) => {
-      const expanded = this.state.activeItems.indexOf(index) !== -1;
+    const children = arrayify(this.props.children);
+    return children.map((item, index) => {
```

Comment:
If the purpose of `arrayify` is to allow you to map through the array, does it make sense to use something like this
```javascript
const { children } = this.props;

return Object.keys(children).map(index => {
const key = children[index].props.slug || index;
...
});
```

Kind of feels as though arrayify could be simplified, but not sure whether I have enough context